src/Reducer.js

```diff
@@ -1,7 +1,7 @@
 import sets from './json/sets.json';
 import reducers from './actions';
 
-const initialState = {setList: sets, cardList: [], userObject: {}, isUserLoggedIn: true};  
+const initialState = {setList: [sets], setSpecificCardList: [], cardQueue: [], userObject: {}, isUserLoggedIn: false};  
```

Comment:
now that sets is coming from an on mount, It should be an empty array and the JSON import removed.